modules/microshift-ca-adding-bundle-ostree.adoc

```diff
@@ -0,0 +1,53 @@
+//Module included in the following assemblies:
+//
+//* microshift_install/microshift-embed-in-rpm-ostree.adoc
+
+
+:_content-type: PROCEDURE
+[id="microshift-ca-adding-bundle-ostree_{context}"]
+= Adding a certificate authority bundle to an ostree image
+You can include additional trusted certificate authorities (CAs) to the {op-system-ostree-first} `rpm-ostree` image by adding them to the blueprint you use to create the image. Using the following procedure sets up additional CAs to be trusted by the operating system when pulling images from an image registry.
+
+[NOTE]
+====
+The procedure requires you to configure the CA bundle customizations in the blueprint, then add steps to your kickstart file to enable the bundle. In the following steps, `data` is the key, `<value>` represents the PEM-encoded certificate.
+====
+
+.Prerequisites
+
+* You have root-user access to your build host.
+* Your build host meets the Image Builder system requirements.
+* You have installed and set up Image Builder and the `composer-cli` tool.
+
+.Procedure
+
+. Add the following custom values to your blueprint to add a directory, `/etc/pki/ca-trust/source/anchors/`, and a file, `/etc/pki/ca-trust/source/anchors/cert1.pem`, to the image:
+
+.. Add instructions to your blueprint to create the directory for your certificate bundles:
++
+[source,terminal]
+----
+[[customizations.directories]]
+path = "/etc/pki/ca-trust/source/anchors"
+----
+
+.. Add instructions to your blueprint to create the certificate bundles:
++
+[source,terminal]
+----
+[[customizations.files]]
+path = "/etc/pki/ca-trust/source/anchors/cert1.pem"
+data = "<value>"
+----
+
+. To enable the certificate bundle in the system-wide trust store configuration, use the `update-ca-trust` command, for example:
++
+[source,terminal]
+----
+$ sudo update-ca-trust
```

Comment:
The first step, modifying the blueprint, has to happen on the host where the image is being built.

This step, updating the trust bundle, needs to happen on the host where that image has been booted.

Can can we express that separation without turning this into a much longer procedure?

@ggiguash are we sure we have to explicitly invoke update-ca-trust?

Reply:
According to my previous tests with image mirroring, the `update-ca-trust` command should run for the certificates to become available in the default store. 

You're right that we need to be more explicit on where to run these commands.

Reply:
@ggiguash @dhellmann I added some clarification to these steps in the most minimal way I could think of. My only concern is I am wondering if we need one more step at the beginning like "Create an image and push to a registry"   or a prerequisite step that says something like "you have an image building on your host" I am not sure if that would apply here or if my thoughts are even correct.

Reply:
The new steps about the blueprint are extensions of the instructions we have for building the ostree image in the installation chapters. Maybe we should reference that? Or add those 2 steps as optional there? I'm not sure what we would do about running update-ca-trust, though. Do we talk about kickstart files in that section?

Reply:
Done! Thanks!